### PR TITLE
fix creation customer link (even if option SOCIETE_DISABLE_PROSPECTSCUSTOMERS is set)

### PR DESCRIPTION
**Pull request by ghost**
fix creation customer link (even if option SOCIETE_DISABLE_PROSPECTSCUSTOMERS is set)

### CODE REVIEW

File: htdocs/core/menus/standard/eldy.lib.php [hunk 1]
```diff
@@ -648,7 +648,7 @@ function print_left_eldy_menu($db,$menu_array_before,$menu_array_after,&$tabMenu
 				if ($usemenuhider || empty($leftmenu) || $leftmenu=="prospects") $newmenu->add("/societe/list.php?type=p&amp;sortfield=s.datec&amp;sortorder=desc&amp;begin=&amp;search_stcomm=2", $langs->trans("LastProspectContactInProcess"), 2, $user->rights->societe->lire);
 				if ($usemenuhider || empty($leftmenu) || $leftmenu=="prospects") $newmenu->add("/societe/list.php?type=p&amp;sortfield=s.datec&amp;sortorder=desc&amp;begin=&amp;search_stcomm=3", $langs->trans("LastProspectContactDone"), 2, $user->rights->societe->lire);
 				*/
-				$newmenu->add("/societe/card.php?leftmenu=prospects&amp;action=create&amp;type=p", $langs->trans("MenuNewProspect"), 2, $user->rights->societe->creer);
+				$newmenu->add("/societe/card.php?leftmenu=prospects&amp;action=create&amp;client=2", $langs->trans("MenuNewProspect"), 2, $user->rights->societe->creer);
```
Comment (eldy) [inline htdocs/core/menus/standard/eldy.lib.php:651]: if SOCIETE_DISABLE_PROSPECTSCUSTOMERS is not set (so default setup) if you use client=2, you will have form preselected on customer only when we want a default value to be preselected on prospect+customer

Comment (ghost) [reply htdocs/core/menus/standard/eldy.lib.php:651]: With or without this option, when we click on "new prospect" into menu, the field type should be preselected to prospect and client on "new customer " the field should be preselected to customer.
Should I define this default "new customer" preselected type into setup (for Dolibarr 10 then) ?

Comment (ghost) [reply htdocs/core/menus/standard/eldy.lib.php:651]: may be this #10514 ?

**Comment by atm-maxime** [comment]
Why new customer menu should prefill the form with 'prospect/customer' and not just 'customer' ? What is the use of 'prospect/customer' anyway ?

**Comment by ghost** [comment]
I close as you merge PR on dev branch

**Pull request closed without merging**